pgjdbc/src/test/java/org/postgresql/jdbc/ScramTest.java

```diff
@@ -63,7 +63,7 @@ void testPasswordWithSpace(String passwd) throws SQLException {
     createRole(passwd); // Create role password with spaces.
 
     Properties props = new Properties();
-    props.setProperty("username", ROLE_NAME);
+    props.setProperty("user", ROLE_NAME);
```

Comment:
Nice catch! could you please change this to:
```java
PGProperty.USER.set(props, ROLE_NAME);
PGProperty.PASSWORD.set(props, passwd);
```

Reply:
of course , I'll change it